swhkd/src/daemon.rs

```diff
@@ -43,12 +43,34 @@ impl KeyboardState {
     }
 }
 
+/// Simple Wayland Hotkey Daemon
+#[derive(Parser)]
+#[command(version, about, long_about = None)]
+struct Args {
+    /// Set a custom config file path.
+    #[arg(short, long, value_name = "FILE")]
+    config: Option<PathBuf>,
+
+    /// Set a custom repeat cooldown duration. Default is 250ms.
+    #[arg(short, long)]
```

Comment:
Since both `cooldown` and `config` have similar long and short options, it seems to cause a panic

<details>
<summary>Exact Panic</summary>

thread 'main' panicked at /home/noobscience/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.5.1/src/builder/debug_asserts.rs:112:17:
Command Simple-Wayland-HotKey-Daemon: Short option names must be unique for each argument, but '-c' is in use by both 'config' and 'cooldown'
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

</details>

Just removing the short option for either one of them would fix the error I think

Also, similarly for `debug` and `device` options as well :)

Reply:
Fixed in the latest commit.